agrirouter-api-java-api/src/main/kotlin/com/dke/data/agrirouter/api/dto/revoke/RevokeRequest.kt

```diff
@@ -0,0 +1,17 @@
+package com.dke.data.agrirouter.api.dto.revoke
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class RevokeRequest {
+
+    lateinit var accountId: String
+
+    lateinit var endpointIds: Array<String>;
+
+    lateinit var UTCTimestamp: String
```

Comment:
Please use valid naming. Uppercase variable names aren't a good choice.

Reply:
Disagree. The class is used to map a JSON Object required by agrirouter. if we change the case, we break the JSON structure, which is case sensitive. See https://github.com/DKE-Data/agrirouter-interface-documentation/blob/develop/docs/integration/revoke.adoc

Reply:
To map the fields correctly there are annotations for each framework. See `SendMessageRequest` for example. Therefore this should not be a problem.

Reply:
Adjusted